src/sparseml/pytorch/optim/modifier.py

```diff
@@ -407,6 +417,9 @@ class ScheduledModifier(Modifier, BaseScheduled):
     :param start_epoch: The epoch to start the modifier at
     :param end_epoch: The epoch to end the modifier at
     :param log_types: The loggers that can be used by the modifier instance
+    :param log_frequency: The number of epochs or fraction of epochs to
```

Comment:
Since we can set `log_frequency` to a full number of epochs or alternatively a fraction of epoch, this allows us to log on the "epoch" timescale as well as  "training step" timescale right? By timescale I mean temporal granularity, if you will - what we have discussed during our ML Eng meeting.

Reply:
Yep! You can set it to any positive float to achieve your desired granularity